pylint/checkers/imports.py

```diff
@@ -881,20 +881,19 @@ def _check_import_as_rename(
             if not all(name):
                 return
 
-            real_name = name[0]
-            splitted_packages = real_name.rsplit(".", 1)
-            real_name = splitted_packages[-1]
-            imported_name = name[1]
-            if real_name != imported_name:
+            splitted_packages = name[0].rsplit(".", maxsplit=1)
+            import_name = splitted_packages[-1]
+            aliased_name = name[1]
+            if import_name != aliased_name:
```

Comment:
@cdce8p does this look ok?

Reply:
i realised that i should have named it 

``consider-using-from-import`` instead of ``consider-use-from-import`` -- let me fix that